src/app/components/ms-item/ms-item.component.scss

```diff
@@ -0,0 +1,22 @@
+.section {
+    margin-bottom: 20px;
+}
+
+.col-1 {
+    max-width: 7%;
```

Comment:
Avoid using the percentage value if it's not necessary. Bootstrap has default utility classes the might be usefull in this cases https://getbootstrap.com/docs/4.1/utilities/spacing/